app/assets/stylesheets/views/_landing_page.scss

```diff
@@ -22,3 +22,23 @@
     text-decoration: none;
   }
 }
+
+// Consistent lower margins for top-level blocks
+// Self-chaining the class name gives higher
+// specificty and avoids the use of !important.
+//
+// Note the desktop size is hardcoded here as the
+// govuk-spacing(x) helper does not include large
+// enough sizes.
+
+.app-margin-bottom-responsive.app-margin-bottom-responsive  {
```

Comment:
I think keeping `app` in the name is prudent since it helps identify where the style is coming from: that's useful for other developers who might be looking at the markup to obtain clues for where to find associated styles. I don't really want to use the word `block` here because it might cause confusion over whether we're talking about the blocks of content or block level margins. On balance then, I think the original class name I proposed makes more sense.

Reply:
Using "app" is not a pattern we use anywhere but components, though, let's avoid that.

Reply:
OK, understood. I think I'll revisit this once I've nailed the approach: if we end up using a mixin we might not need this named class at all.